XML Resume File

```diff
@@ -27,24 +27,43 @@ Maintain equipment inventory, accurately record asset details in Google Spreadsh
 Generate Asset Hand receipts for yearly inventory and research asset history using Sunflower inventory system.</DUTIES>
    </EXPERIENCE>
   <EXPERIENCE>
-    <COMPANYNAME></COMPANYNAME>
-    <COMPANYCONTACTINFORMATION Preferred="URL"</COMPANYCONTACTINFORMATION>
-    <EMPLOYMENTDATES></EMPLOYMENTDATES>
-    <TITLE></TITLE>
-    <DUTIES></DUTIES>
+    <COMPANYNAME>Ipswich Public Schools</COMPANYNAME>
+    <COMPANYCONTACTINFORMATION Preferred="https://www.ipsk12.net/Page/9"</COMPANYCONTACTINFORMATION>
+    <EMPLOYMENTDATES>August 2013-October 2015</EMPLOYMENTDATES>
+    <TITLE>Technical Specialist</TITLE>
+    <DUTIES>Technical Team member for 1.5 Million dollar large scale IT infrastructure upgrade for the Ipswich Public School System. Assisted with all aspects of modernizing, building and configuring a new managed network including planning, and overseeing installation and implementation of the IT infrastructure improvement plan.
+Data migration coordinator for district cut over from MMS SIS to Aspen SIS.
+      
+Aspen PreK-12 Student Information System Administrator.
+      
+Created and maintained district inventory database where none existed. Added items to database, assigned numbers, generated barcodes and used handheld barcode reader to add to database.
+Provided IT Helpdesk support to 2000 end users.
+      
+Created and maintained organizational system in IT set room and server room. Created server room and network map.</DUTIES>
```

Comment:
@DCARSENAULT  this all looks great.  only comment is that for sections like this - xml systems won't process line breaks. What you can do is within <DUTIES>, add "child" elements, such as <para> and put each duty in a separate  <para> element. That way, whatever processing happens downstream to create a document, it will know  that they are separate, and not run them together in one long line.

Reply:
Thanks, Fixed.